projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -708,10 +708,10 @@ public Dictionary<string, string> PlaceOfBirth
             {
                 try
                 {
-                    if (!String.IsNullOrEmpty(value["addressState"]) && value["addressState"] != "TT" && !CodeExistsInValueSet(value["addressState"], VR.ValueSets.Jurisdiction.Codes))
-                    {
-                        return;
-                    }
+                    // if (!String.IsNullOrEmpty(value["addressState"])  && !CodeExistsInValueSet(value["addressState"], VR.ValueSets.Jurisdiction.Codes))
+                    // { 
+                    //     return;
+                    // }
```

Comment:
Delete commented out code

Reply:
By removing this check, are we saying that the library will not enforce what jurisdiction values are allowed for the child, fetus, or mother place of birth fields and that is up to NCHS to check in their business rules?

Reply:
TT, TS, and ZZ are incorporated into the valueset that gets checked for every set.  So, there is no longer special handling for these values.   TT, TS, and ZZ are (will be) conformant with the IG, and NCHS/STEVE/Jurisdictions will need their own business rules to decide when these values are acceptable as a jurisdiction.  There is no way to code that into a library that supports production and testing.   Currently, the code in SetPlaceOfBirth just sets a dictionary, it doesn't check the contents.  I think that is the correct approach.

The comment has been deleted.